terraform-infra-approvals/bulk-scan-shared-infrastructure.json

```diff
@@ -4,6 +4,7 @@
     {"type": "azurerm_network_security_group"}
   ],
   "module_calls": [
-   {"source":  "git@github.com:hmcts/cnp-module-palo-alto?ref=test-pip-upgrade"}
+   {"source":  "git@github.com:hmcts/cnp-module-palo-alto?ref=test-pip-upgrade"},
+   {"source":  "git@github.com:hmcts/cnp-module-waf?ref=CHG5001024"}
```

Comment:
what's the plan for this is this a temp change?

afaik it's only bulk-scan, idam and XUI still using this module, XUI are going live on wednesday on AKS and will stop using the module then, you could probably just mainline the change?

Reply:
We're going to move Bulkscan over to the DMZ/Hub eventually. I'm just speaking with @nitinprabhu now about our approach, but hopefully we can do it as part of the next PI. So realistically expect this to live for no more than a few months.

Reply:
We want to keep this standalone as we don't want to introduce any policy changes to XUI or IDAM and we don't see this living long enough to justify effort to mainlining.